wandb/proto/wandb_internal.proto

```diff
@@ -424,6 +424,11 @@ message ArtifactManifestEntry {
   string birth_artifact_id = 7;
   // Whether to avoid copying/moving files to the cache while uploading.
   bool skip_cache = 8;
+  enum Policy {
+    MUTABLE = 0;
```

Comment:
By using a 0, "MUTABLE" is the default if the policy field is unset. Just double checking that this is intentional.

A common practice is to use a special zero value, e.g. "ARTIFACT_POLICY_UNSET", and let the handling code control what the default behavior is. You can never change proto field numbers, but you can always go back and change the Python code.

Reply:
Default => "Mutable" is intentional